Data.py

```diff
@@ -29,10 +29,12 @@ def __init__(self):
                                        self.hdf5_runs_path)
 
         # Load data indexes for training and validation
+        self.train_all_steer_path = ARGS.data_path + '/train_all_steer'
```

Comment:
Will do. I wasn't sure either but I followed the pattern of self.hdf5_runs_path.